test/test_filter_masking.rb

```diff
@@ -17,6 +17,12 @@ def setup
   # default configuration for tests
   CONFIG = %[
     fieldsToMaskFilePath test/fields-to-mask
+    fieldsToExcludeJSONPaths excludeMaskFields,exclude.excludeMaskFields
```

Comment:
```suggestion
    fieldsToExcludeJSONPaths excludedField,exclude.path.nestedExcludedField
```

makes the names a bit more understandable + test here a multiple nested field